### DISCUSSION TITLE
IndexOptimize elapsed time in ssms different from "sum" in CommandLog

### DISCUSSION THREAD

**Discussion by pasqualeceglie** [olahallengren/sql-server-maintenance-solution#459] [Questions & Answers] [answered]
Hi Ola, I've executed IndexOptimize on my db in a new blank query in SSMS. There is a consistent difference between the execution time in the query window and the sum of elapsed time (difference between the Endtime and StartTime column for every row). 
How is it possible?

Thanks,

**Comment by olahallengren** [comment, marked as answer]
It is only the ALTER INDEX and UPDATE STATISTICS commands, that are logged to the CommandLog table.

The checks that are performed before these commands (e.g. querying sys.dm_db_index_physical_stats), are not included.
